x-pack/plugins/maps/server/maps_telemetry/maps_telemetry.test.js

```diff
@@ -74,73 +74,136 @@ describe('buildMapsSavedObjectsTelemetry', () => {
   test('returns zeroed telemetry data when there are no saved objects', async () => {
     const result = buildMapsSavedObjectsTelemetry([]);
 
-    expect(result).toMatchObject({
-      attributesPerMap: {
-        dataSourcesCount: {
-          avg: 0,
-          max: 0,
-          min: 0,
-        },
-        emsVectorLayersCount: {},
-        layerTypesCount: {},
-        layersCount: {
-          avg: 0,
-          max: 0,
-          min: 0,
-        },
+    expect(result.layerTypes).toEqual({});
+    expect(result.scalingOptions).toEqual({});
+    expect(result.joins).toEqual({});
+    expect(result.basemaps).toEqual({});
+    expect(result.attributesPerMap).toEqual({
+      dataSourcesCount: {
+        avg: 0,
+        max: 0,
+        min: 0,
+      },
+      emsVectorLayersCount: {},
+      layerTypesCount: {},
+      layersCount: {
+        avg: 0,
+        max: 0,
+        min: 0,
       },
-      mapsTotalCount: 0,
     });
+    expect(result.mapsTotalCount).toEqual(0);
+    expect(new Date(Date.parse(result.timeCaptured)).toISOString()).toEqual(result.timeCaptured);
```

Comment:
Is this comparison really helpful?

Reply:
tried to introduce a lightweight check to ensure the time-field is added (and it is a valid time-field). didn't want to introduce moment-js for this.